sql/core/src/test/java/test/org/apache/spark/sql/JavaHigherOrderFunctionsSuite.java

```diff
@@ -222,10 +222,10 @@ public void testTransformKeys() throws Exception {
         checkAnswer(
             mapDf.select(transform_keys(col("x"), (k, v) -> k.plus(v))),
             toRows(
-                CollectionConverters.MapHasAsScala(new HashMap<Integer, Integer>() {{
+                CollectionConverters.asScala(new HashMap<Integer, Integer>() {{
```

Comment:
@LuciferYang can you find other similar places and use this proper/simpler API instead?

Reply:
Does it only affect Java? This might be the only case